packages/iris-grid/src/AdvancedFilterCreatorSelectValueList.tsx

```diff
@@ -140,7 +140,7 @@ class AdvancedFilterCreatorSelectValueList<T = unknown> extends PureComponent<
     );
     if (selectedIndex >= 0) {
       selectedValues.splice(selectedIndex, 1);
-    } else {
+    } else if (value != null) {
```

Comment:
This changes the logic of this function. Need to test what happened if the value was null before to ensure we shouldn't be doing something else in that case or typing `selectedValues` differently

Reply:
It seems like it should be possible to push `null` since `makeAdvancedFilter` will be called on this array in `AdvancedFilterCreator.tsx` and it should be possible to filter on a `null` value. I changed the typing of `selectedValues` to `T | null` to get rid of the TS errors.